doc/getting-started.md

```diff
@@ -1,5 +1,9 @@
 # Getting Started with Ignition #
 
+## What is Ignition? ##
+
+Ignition is a boot-time configuration utility that runs before the switch out of the initramfs during boot.  Mechanically-speaking, it is an executable in the initramfs that looks for configuration data (or a pointer to a URL for such data) in an environment-specific location and applies it to the machine before switch_root is called (see below under [Providing a Config](#providing-a-config) for details).
```

Comment:
Ignition is a low-level system configuration utility. The Ignition executable is part of the temporary initial root filesystem, the *initramfs*. When Ignition runs, it finds configuration data in a named location for a given environment, such as a file or URL, and applies it to the machine before `switch_root` is called to mount the machine's root filesystem.